src/main/java/bwapi/ClientData.java

```diff
@@ -1,1993 +1,2000 @@
-package bwapi;
```

Comment:
Autogenerated file. I probably changed line endings when I regenerated it. Give a 👍 if you'd like to see fixed line endings.